picturae_importer.py

```diff
@@ -91,11 +91,11 @@ def init_all_vars(self):
         self.batch_md5 = generate_token(starting_time_stamp, self.file_path)
 
         self.record_full['batch_md5'] = self.batch_md5
-
-        self.monitoring_tools = MonitoringToolsDir(config=self.picturae_config,
-                                                   batch_md5=self.batch_md5,
-                                                   report_path=self.picturae_config.ACTIVE_REPORT_PATH,
-                                                   active=True)
+        if self.picturae_config.MAILING_LIST:
+            self.monitoring_tools = MonitoringToolsDir(config=self.picturae_config,
+                                                       batch_md5=self.batch_md5,
+                                                       report_path=self.picturae_config.ACTIVE_REPORT_PATH,
+                                                       active=True)
 
```

Comment:
Yeah, we have four of these code blocks, If we can push all this logic down a level everything cleans up nicely.